src/status_im/components/styles.cljs

```diff
@@ -19,6 +19,7 @@
 (def color-gray5 "#d9dae1")
 (def color-gray6 "#212121")
 (def color-gray7 "#9fa3b4")
+(def color-gray8 "#6E777E")
```

Comment:
Wow 8 different grays :)

Reply:
#TooManyGraysException

Reply:
tbh i don't know what to do with that. and i don't like it, we need some styles sheet from designers with color names

Reply:
:+1: